app/src/main/java/com/fer1592/k8s_android_console/viewmodel/ClustersViewModel.kt

```diff
@@ -28,7 +35,9 @@ class ClustersViewModel(private val clusterRepository: ClusterRepository = Clust
     // Function that deletes a cluster
     fun deleteCluster(cluster: Cluster) {
         viewModelScope.launch(dispatcher) {
+            _processingData.postValue(true)
             clusterRepository.deleteCluster(cluster)
```

Comment:
what if the cluster deletions fail? like if we set an inexistent cluster in the DB? Maybe we should wait for the deletion result and show a message to the user

Reply:
I show a message in case the delete succeeds or fails. It was out of the scope of "animations" but I'll everything to make you happy